packages/components/package.json

```diff
@@ -28,7 +28,7 @@
     "history": "^4.9.0",
     "lodash": "^4.17.19",
     "path-to-regexp": "^1.7.0",
-    "react-colorful": "4.1.1",
+    "react-colorful": "4.4.4",
```

Comment:
This gives us better a11y interactions (keyboard).
I think better performance/reliability as well.